### ISSUE TITLE
bert-base-multilingual-uncased vocabulary not consecutive

### ISSUE THREAD

**Issue by ntubertchen** [huggingface/transformers#990]
## 🐛 Bug

When I was checking out bert-base-multilingual-uncased vocabulary. I receive the warning "Saving vocabulary to ./vocab.txt: vocabulary indices are not consecutive. Please check that the vocabulary is not corrupted"

I ran the similar command on two different machine and got the same warning.

from pytorch_transformers import *
tokenizer = BertTokenizer.from_pretrained('bert-base-multilingual-uncased',do_lower_case=True)
tokenizer.save_vocabulary('./')

I ran it on 

* OS:
* Python version: python3.5 
* PyTorch version: pytorch1.0.1.post2
* PyTorch Transformers version (or branch): 1.0
* Using GPU ? Yes
* Distributed of parallel setup ?no

**Comment by LysandreJik** [comment]
H! Could you please specify on which OS you have this error? I cannot reproduce this on MacOS 10.15, nor on Ubuntu 18.04 with both Python 3.5 and 3.6.

**Comment by ntubertchen** [comment]
Hi, @LysandreJik 

I tried these on Ubuntu 16.04.6 LTS.

**Comment by gilnoh** [comment]
Just in case helpful for you -- I had exactly the same issue with release 1.0 of pytorch-transformer. What ever I do with multilingual model, it gave me the above warning. And the results were quite strange. (e.g. much lower than expected). I installed master branch (say, 1.1 in working) today and the error went away. 

Try install from current master (which will install locally built 1.1); it helped me. I think some code has been changed on handling multilingual vocab file.
